.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-  - 0.8
\ No newline at end of file
+  - 0.12
```

Comment:
but node version 0.8 is no longer maintained - https://github.com/nodejs/LTS#lts-schedule